tests/main/tests/unit/record-arrays/record-array-test.js

```diff
@@ -141,6 +141,49 @@ module('unit/record-arrays/record-array - DS.RecordArray', function (hooks) {
     }
   );
 
+  deprecatedTest(
+    '#rejectBy',
+    { id: 'ember-data:deprecate-array-like', until: '5.0', count: 3 },
```

Comment:
not sure what value `count` should be.

Reply:
count is number of expected times this deprecation will be seen